README.md

```diff
@@ -206,3 +206,247 @@ Check logs across pods using `kc_logs.sh` script from
 ```bash
 kc_logs.sh razee razeedash-api 1m
 ```
+
+## Web hooks
+
+Implemented web hooks so data in Razeedash can be augmented by third-party
+services such as test suites and vulnerability scanners.
+
+Components are:
+
+- Web hook creation
+- Web hook deletion
+- Trigger by resource ID
+- Trigger by Regex
+- Callback response
+
+### Web hook definition
+
```

Comment:
Do we have or need a `GET` to allow the UI to get the registered webhooks for management?

Reply:
We could add it for completeness but I would imagine the UI will directly access mongoDB.